addon/model.js

```diff
@@ -1,3 +1,6 @@
+// This lint error disables "this.attrs" everywhere.  What could go wrong?
+/* eslint-disable ember/no-attrs-in-components */
```

Comment:
Can you file an issue over on eslint-plugin-ember for this? It seems pretty wild that this lint failure is biting us in a completely unrelated file.

Also, I'd prefer to disable this rule in the `.eslintrc.js` file, instead of inline like this.

Reply:
Will file an issue. We already defined this in other files inline, so keeping with that. But happy to change wholesale as a separate PR